benchmarks/memory/Dockerfile

```diff
@@ -1,12 +1,26 @@
 FROM node:14-buster
+ARG jemalloc
 ENV NODE_ENV=production
 ENV CI=1
 ENV GATSBY_CPU_COUNT=4
 RUN apt-get update -y && apt-get upgrade -y && apt-get install git curl npm -y
 RUN npm i -g gatsby-cli gatsby-dev-cli
-WORKDIR /usr/src/app
+
+# set heap to 16gb just to catch all test cases
+ENV NODE_OPTIONS="--max-old-space-size=16368"
+
 RUN echo "\n\necho \"Welcome to the Gatsby Memory benchmark container!\\n  - /usr/src/gatsby : Your local gatsby repo\\n  - /usr/src/app : The memory benchmark gatsby site\\n\"" > /root/.bashrc
 
+RUN if [ "$jemalloc" = "1" ]; then \
+  echo "Using jemalloc for memory allocation" && \
+  apt-get update && apt-get install -y libjemalloc-dev=5.1.0-3 && \
+  echo "/usr/lib/x86_64-linux-gnu/libjemalloc.so" >> /etc/ld.so.preload && \
```

Comment:
Not blocking, just future opportunity:

Maybe we can just install `jemalloc` always in image and then have `--use-jemalloc` as toggle for cli (we could set `LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so` env var in test runner script to enable it), this could limit how often we have to (re)build image when testing

Reply:
Good call. I think if we revisit this, that's a good improvement.